### PR TITLE
v1.16.3 release

### PR DESCRIPTION
**Pull request by anthonybilinski**
- [x] Commits follow our [git commit guidelines](https://github.com/qTox/qTox/blob/master/CONTRIBUTING.md#git-commit-guidelines)

---
This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/5245)

### CODE REVIEW

**Comment by sudden6** [comment]
Not sure how to make this build on CI, but it lgtm

**Comment by Diadlo** [comment]
@sudden6 Probably not merge in master. But IDK how. Maybe: Create PR to some branch, add and push tag, remove branch

**Comment by anthonybilinski** [comment]
I tried locally pushing a new divergent tag to a different local repo both on master and it worked like I wanted, so going to give that a try here. Since github doesn't support PRing tags and they're both on master, I don't think it can understand what we're trying to do =/

**Comment by anthonybilinski** [comment]
It worked, looks good on master, going to close PR. Maybe this is the "correct" way to do this, though it looks like we haven't made a patch off-tip since 1.7.1.

**Comment by anthonybilinski** [comment]
flatpak build is failing due to patch rejection, even though it appears to have worked for 1.16.1 release and there have been no changes to the flatpak build system. I think our io.github.qtox.qTox.json stored on github has been updated, breaking release flatpak build for 1.16.2, but I also can't build 1.16.1 flatpak locally for the same reason.

I haven't been following flatpak very closely, but apparently we need to keep hosting all versions of flatpak json files indefinitely and tie qTox versions to the flatpak json files so that any flatpak version remains buildable into the future.

@sudden6 is this possible to fix without code changes by updating the flatpak json, or does the release need to be updated / bumped?

**Comment by sudden6** [comment]
#5236 should fix that

**Comment by anthonybilinski** [comment]
It does not fix it. The problem is that the patch is trying to apply on top of version information:
```
<                   "type": "git",
<                   "url": "https://github.com/qTox/qTox",
<                   "tag": "v1.15.0", 
<                   "commit": "02d6c63acaac0ae95fa8be3a1b9301657e6a4a94"
<               }
<           ]
```
but the current file at https://raw.githubusercontent.com/flathub/io.github.qtox.qTox/master/io.github.qtox.qTox.json gets updated once per qTox release. So we need some kind of synchronization between updating the patch and updating the flathub manifest, or pointing our flatpak build to a specific commit/tag/release of our flathub manifest instead of master tip.

**Comment by sudden6** [comment]
sorry, linked the wrong stuff, aabdb500861727f63bb8f5c8551313a4a6b2c3ae is the fix and it's already in master.

**Comment by anthonybilinski** [comment]
Added aabdb500861727f63bb8f5c8551313a4a6b2c3ae as well as current PR https://github.com/qtox/qtox/pull/5236, and ran CI on my branch this time, passed and built flatpak successfully: https://travis-ci.org/anthonybilinski/qTox/jobs/405210974

I bumped version to 1.16.3 since the 1.16.2 tag was already pushed to master =/

With the flatpak manifest being in our repo, there's the comment "but has the downside that we have to sync to the flathub.org repo on a release". How is this done? Can it be added to https://github.com/qTox/qTox/blob/master/MAINTAINING.md?

File: flatpak/io.github.qtox.qTox.json [hunk 1]
```diff
@@ -0,0 +1,157 @@
+{
+    "app-id": "io.github.qtox.qTox",
+    "runtime": "org.kde.Platform",
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.10",
```
Comment (sudden6) [inline flatpak/io.github.qtox.qTox.json]: I think we should update to 5.11 here, in order to fix https://github.com/flathub/io.github.qtox.qTox/issues/3

**Comment by sudden6** [comment]
I have added instructions on how to update Flathub in #5250, you also should have write permissions for the repository now.

**Review by anthonybilinski — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/5245)* status: 0 of 1 LGTMs obtained

___
*[flatpak/io.github.qtox.qTox.json, line 5 at r4](https://reviewable.io/reviews/qtox/qtox/5245#-LHgMN-YBD1R0PN0ubsp-r4-5:-LHtWRqiES6cbDESv_c-:b-896fix) ([raw file](https://github.com/qtox/qtox/blob/9a78fc26db64c41ec6791baefbdc03654e7adfce/flatpak/io.github.qtox.qTox.json#L5)):*
<details><summary><i>Previously, sudden6 wrote…</i></summary><blockquote>

I think we should update to 5.11 here, in order to fix https://github.com/flathub/io.github.qtox.qTox/issues/3
</blockquote></details>

Done.

**Comment by anthonybilinski** [comment]
Travis passing on my branch (un-merged to master): https://travis-ci.org/anthonybilinski/qTox/jobs/406385303

**Comment by sudden6** [comment]
lgtm

**Pull request merged**